assignments/tasks_test.go

```diff
@@ -21,10 +21,10 @@ type foundIssue struct {
 
 // The test environment creates tasks based on the issues found on student repositories in the organization.
 // This is so that we can emulate the result of a previous push to the tests repository.
-// For these tasks to be created correctly the title of each task must use the following format:
+// For these tasks to be created correctly the title of each issue must use the following format:
```

Comment:
This requirement seems unnecessary to me. Can't we use the string `lab1/hello_world` or just `lab1` to lookup the assignment order from somewhere? (Didn't check)

Reply:
No longer valid